docs/consume-packages/installing-signed-packages.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Install a signed NuGet package
+description: Describes the process of installing signed NuGet packages and configuring package signature trust settings.
+author: karann-msft
+ms.author: karann
+ms.date: 11/29/2018
+ms.topic: conceptual
+---
+
+# Install a signed package
+
+Signed packages don't require any specific action to be installed; however, if the content has been modified since it was signed, the installation is blocked with error [NU3008](../reference/errors-and-warnings/NU3008.md).
+
+> [!Warning]
+> Packages signed with untrusted certificates are considered as unsigned and are installed without any warnings or errors like any other unsigned package.
+
+## Configure package signature requirements
+
+*NuGet 4.9.0+ and Visual Studio version 15.9 and later on Windows*
+
+You can configure how NuGet clients validate package signatures by setting the `signatureValidationMode` to `require`. 
+
+```xml
+  <config>
+    <add key="signatureValidationMode" value="require" />    
+  </config>
+```
+
+This mode will verify that all packages are signed by any of the certificates trusted in the `nuget.config` file. This file allows you to specify which authors and/or repositories are trusted based on the certificate's fingerprint. 
+
+### Trust package author
+
+To trust packages based on the author signature use the `authors` element:
```

Comment:
The element is `author` in the example on line 37. I would change this to `author`.

Reply:
done